### PR TITLE
triage: add more `long build` formulae

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
need to get [this PR](https://github.com/Homebrew/actions/pull/257) merged first before refactoring the code style in here.

**Review by dawidd6 — DISMISSED** [review]

File: .github/workflows/triage.yml [hunk 1]
```diff
@@ -135,7 +135,7 @@ jobs:
               content: system "swift", "build"
 
             - label: long build
-              path: Formula/(agda|arangodb|boost|deno|dotnet|emscripten|gcc|ghc|llvm|node|ponyc|rust|swift|texlive|qt)(@[0-9]+)?.rb
+              path: Formula/(agda|arangodb|boost|cmake|deno|dotnet|emscripten|gcc|ghc|llvm|node|pango|ponyc|rust|suite-sparse|swift|texlive|qt)(@[0-9]+)?.rb
```
Comment (carlocab) [inline .github/workflows/triage.yml:138]: Just a note here: I was deliberately conservative about adding formulae to this list. In particular, I left out formulae that by themselves (without dependents) would not require the longer timeout.

We can change that approach, of course, but I just wanted to be sure that that's what we want to do here.

Comment (Bo98) [reply .github/workflows/triage.yml:138]: I'd rather `cmake` was labelled to have `CI-skip-recursive-dependents` to be honest (or better: make test-bot not follow test dependencies recursively - I'll need to look into that at some point).

Comment (carlocab) [reply .github/workflows/triage.yml:138]: > make test-bot not follow test-only dependencies recursively

I've thought about this too. Should this not be similar to how we don't follow build-only dependencies recursively?

Comment (Bo98) [reply .github/workflows/triage.yml:138]: Yes, I think so.

Comment (MikeMcQuaid) [reply .github/workflows/triage.yml:138]: Agreed with both of you, sounds like we're OK with `cmake` being removed and the rest included.
```suggestion
              path: Formula/(agda|arangodb|boost|deno|dotnet|emscripten|gcc|ghc|llvm|node|pango|ponyc|rust|suite-sparse|swift|texlive|qt)(@[0-9]+)?.rb
```

**Review by MikeMcQuaid — COMMENTED** [review]
Good once `cmake` is removed!

**Review by SMillerDev — DISMISSED** [review]

**Comment by chrmoritz** [comment]
Maybe you could add `v8` to that list too: #99848

**Comment by chenrui333** [comment]
let's create a separate PR for v8, I will do that.

Merging this one now.

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**